wrapper/build.gradle.kts

```diff
@@ -28,7 +28,12 @@ plugins {
 
 dependencies {
     implementation("org.checkerframework:checker-qual:3.26.0")
+    compileOnly("org.apache.httpcomponents:httpclient:4.5.14")
     compileOnly("software.amazon.awssdk:rds:2.20.49")
+    compileOnly("software.amazon.awssdk:sts:2.21.6")
+    compileOnly("software.amazon.awssdk:iam:2.21.6")
+    compileOnly("software.amazon.awssdk:sso:2.21.6")
+    compileOnly("software.amazon.awssdk:ssooidc:2.21.6")
```

Comment:
What are these two for?

Reply:
Ah these were when i was playing with the SSO libraries. I'll remove them.

Reply:
Removed!